stacker_blueprints/rds/aurora/base.py

```diff
@@ -179,6 +184,18 @@ def create_security_group(self):
                 )
             )
             self.security_group = Ref(sg)
+
+            if variables["AllowedCIDRs"]:
```

Comment:
So we tend to do this in a separate stack @ Remind - we found that we wanted to update security rules more often than databases themselves, so it felt safer.  It also meant we could use the full security group rule stack, which can build just about any sort of rule (we tend to use SecurityGroups as the source of the rules, not CIDR).  I think this also gets trickier when you start working with Aurora, right?